### DISCUSSION TITLE
games singe 2

### DISCUSSION THREAD

**Discussion by ghost** [DirtBagXon/hypseus_singe_data#8] [General]
Hello, I would like to know if you will continue to add games 00.monkey2 with the files in the folders because it is really top, and is this principle exists for the games daphne?

**Comment by DirtBagXon** [comment]
As I am made aware of Singe 2 games they should be added. They follow the "folder" format.

Other Singe games are used on SBC's etc and usually do not have the folder structure.

Daphne, does not require a folder structure, the amount, and type, of files in minimal.
